assemble/bin/accumulo-cluster

```diff
@@ -119,6 +119,24 @@ function control_service() {
   host="$2"
   service="$3"
 
+  #
+  # The server processes take arguments (e.g. -a, -p, -o, -q [in the case of the Compactor]).
+  #
+  if [[ $# -gt 3 ]]; then
+    EXTRA_ARGS="${@:4}"
+  else
+    EXTRA_ARGS=""
+  fi
+
+  #
+  # If the host is an IP address, then pass the IP address in the -a argument to the server. If
+  # this is not done, then the default IP address if 0.0.0.0 is assigned and the host name is used
```

Comment:
This is a strange condition on which to decide whether to pass the `-a` or not. `-a` should work regardless of whether it is a hostname or an IP address. This is a longstanding issue with the way we've done our scripts. `-a` is overloaded to mean both "bind address" and "service address to advertise". The check whether it's an IP address seems to imply that `-a` is only used as a bind address. However, a hostname can also refer to a specific NIC, and thus be used for determining which NIC to bind to, and a user may need to specify a hostname to use as the advertisement address, because the IP address it binds to needs to be a local address.

Long-term, we really need to have separate configs for these, and they don't need to be command-line options only, but can actually be configuration properties set in the `accumulo.properties` file.

Short-term, maybe this should always pass the `-a` option, regardless of whether this is an IP address or a hostname?

Reply:
> maybe this should always pass the -a option

I'm good with this